LABTASKS.md

```diff
@@ -1,25 +1,100 @@
+GitHub pages (including changing the line in the `Running your project` section of
+the [README.md](README.md) to link to your team's GitHub pages page).
+
+- [Summary](#summary)
+- [Extending the HTML](#extending-the-html)
+- [Add CSS for the new HTML](#add-css-for-the-new-html)
+- [Create a new CSS file](#create-a-new-css-file)
+- [Technical Requirements for the CSS](#technical-requirements-for-the-css)
+- [How do you know you are "done"? How will you be graded?](#how-do-you-know-you-are-%22done%22-how-will-you-be-graded)
 
 ## Summary
 
-Your task for this lab is to develop two different looks for a web page using
-different CSS without altering the HTML.
-
-- All of the formatting (including the positioning of elements in the page) must
-  be done using CSS.
-- Do not alter the HTML.
-  - The _one_ exception to this rule is if you want to load Google Fonts in your
-    HTML so they'll be available in your CSS. (There are ways to do this entirely in CSS
-    but the "standard" approach typically involves modifying the HTML.)
-  - Be intentional and sparing with your use of absolute sizes for things like font and
-    location. Your style sheet should make the web page look nice for both a phone and
-    a computer screen, and your choices should not interfere with accessibility in
-    obvious ways.
-  - Be sure to follow the setup instructions in the [README.md](README.md) to set up
-    GitHub pages (including changing the line in the `Running your project` section of
-    the [README.md](README.md) to link to your team's GitHub pages page).
-
-## Technical Requirements
+Your tasks for this lab are:
+
+- To add a small bit of HTML using classes and IDs to allow you to format
+  different components in different ways.
+- To develop an additional, new look for a web page using different CSS without altering
+  the HTML.
+
+All of the formatting (including the positioning of elements in the page) must
+  be done using CSS. **Do not alter the HTML** with two exception:
+
+- You have to extend the HTML in the first part. You should do that before you do
+  any of the CSS work.
+- If you want to load Google Fonts you'll want to add those tags to your HTML so
+  they'll be available in your CSS. (There are ways to do this entirely in CSS
+  but the "standard" approach typically involves modifying the HTML.)
+
+Be intentional and sparing with your use of absolute sizes for things like font and
+location. Your style sheet should make the web page look nice for both a phone and
+a computer screen, and your choices should not interfere with accessibility in
+obvious ways.
+
+## Extending the HTML
+
+The current HTML has three major sections:
+
+- CSci Program Student Learning Outcomes (PSLOs)
+- CSci major requirements
+- CSci faculty
+
+You should add a new section to the HTML (without changing any of the existing HTML)
+that provides a list of some of the tools we'll be using in this class:
+
+- [`git`](https://git-scm.com/)
+- GitHub.com
```

Comment:
Shouldn't this have a link like the other items in the list?

Reply:
Good eye. Turns out that GitHub sees this as a URL and automatically turns it into a link when it renders it. Could add an explicit link for consistency, though, if you'd like.

Reply:
I don't think it's important to make it explicit as long as it renders as a URL so they know what URL to use in their html link.

Reply:
When I view labtasks in that branch on GitHub, it does not seem to render it as a link.

Reply:
While it does convert to a link in VS Code's preview, it does not appear to on GitHub.

Reply:
Good catch. I'd assumed that since it was a link in VS Code it would be a link on GitHub. Thanks for fixing that!